### ISSUE TITLE
Make it easier to test rnkiwimobile

### ISSUE THREAD

**Issue by ferrannp** [kiwicom/mobile#1155]
- ~By default use packager (dev vs prod)~ -> Already done https://github.com/kiwicom/mobile/pull/1166
- Docs on how to test native deps

After #1124  & #1160 are merged.

**Comment by ferrannp** [comment]
Done #1166.
